sdks/python/apache_beam/transforms/core.py

```diff
@@ -1448,18 +1448,6 @@ def get_output_type(self):
     return (self.get_type_hints().simple_output_type(self.label) or
             self.infer_output_type(None))
```

Comment:
<!--new_thread; commit:213cff3a659b785a25d6496b8cdbfc292d1fc6e9; resolved:0-->
This is a good line.